### ISSUE TITLE
Completion and linting in the editor for Python files will not work

### ISSUE THREAD

**Issue by mkortink** [spyder-ide/spyder#12032]
### Issue Report Checklist

* [Yes] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [Yes] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [Yes] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ NA] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [Yes] Tried basic troubleshooting (if a bug/error)
    * [Yes] Restarted Spyder
    * [Yes] Reset preferences with ``spyder --reset``
    * [Yes] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [Yes] Tried the other applicable steps from the Troubleshooting Guide
* [Yes] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I have installed Spyder via WinPython(*). When I start Spyder I get the message in the image attached. I have attempted to allow Python through the windows firewall (which is non-trivial in itself!). I still get the problem. 

I don't get this problem with anything else that uses Python including VSCode so I doubt the firewall is involved. 

I note there are some similar issues but they seem to run into trouble after they have been using Spyder for a while. I get the message between clicking to open Spyder and Spyder actually opening. When I cancel the message Spyder opens.

(*) I used to use Anaconda, it is always conflicting with pip, all got too hard, so I have abandoned it. I like Spyder and am tying to keep it at least.

### What steps reproduce the problem?

1.  Install the latest version of WinPython on your Win10 computer and try starting Spyder.
2.  The error may or may not occur for you, but I have fresh installs of everything.

### What is the expected output? What do you see instead?
Spyder starts without an error message and the linter and auto-completion work.

### Paste Traceback/Error Below (if applicable)

Not applicable

```python-traceback
Not applicable

PASTE TRACEBACK HERE
Not applicable

```

## Versions

* Spyder version: 4.1.1
* Python version: 3.8
* Qt version: Don't use it?
* PyQt version: Don't use it?
* Operating System name/version: Win 10 Pro 64bits

### Dependencies

```
PASTE DEPENDENCIES HERE
```

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.2.2 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.10.2 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  None (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.6.7 (OK)
pygments >=2.0                 :  2.5.2 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.6.0 (OK)
qtconsole >=4.6.0              :  4.6.0 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
sphinx >=0.6.6                 :  2.3.0 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.14 (OK)
matplotlib >=2.0.0             :  3.2.0rc2 (OK)
numpy >=1.7                    :  1.17.4 (OK)
pandas >=0.13.1                :  0.25.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5 (OK)
![spyder linter compln](https://user-images.githubusercontent.com/27032470/77834429-d7571e80-7198-11ea-8e1b-08d781030545.PNG)

**Comment by ccordoba12** [comment]
Duplicate of #11026

**Comment by ccordoba12** [comment]
It'll be fixed in our next version.

**Comment by ccordoba12** [comment]
> I get the message between clicking to open Spyder and Spyder actually opening.

This mean that completion and linting won't work for you after you open Spyder. That's not our problem but WinPython's one, so you should report it there.

Our fix will simply avoid showing that message all time, so you can hide it at will.

**Comment by mkortink** [comment]
I get the message in the image I posted telling me linting and completion
won't work, then it doesn't.

**Comment by ccordoba12** [comment]
Ok, sorry. You *could* hide it since our next version.

**Comment by drewsaunders** [comment]
I get exactly the same pop-up in Spyder 4.1.1 under Ubuntu 18.04. I open Spyder, use it for some time without a problem (half hour, say); bu inevitably this pop-up appears, and reappears immediately when I close it. Minimizing it and continuing to work is not an option. I am forced to close and restart Spyder if I want to use it.

**Comment by ccordoba12** [comment]
> I open Spyder, use it for some time without a problem (half hour, say); bu inevitably this pop-up appears, and reappears immediately when I close it. Minimizing it and continuing to work is not an option. I am forced to close and restart Spyder if I want to use it.

We are aware of this problem (see issue #11026 and the multiple complaints there). This will be fixed in 4.1.2, to be released in three or four days.
